### DISCUSSION TITLE
Who uses github-comment?

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [suzuki-shunsuke/github-comment#1162] [Deprecated (General)]
Hi, I'd like to know who uses github-comment.
And I'd like to add the section `Who uses github-comment?` to the document.
If you or your organization uses github-comment, please tell us!

Thank you for your cooperation!

## Reference

- ArgoCD https://github.com/argoproj/argo-cd/blob/master/USERS.md
- tfaction https://github.com/suzuki-shunsuke/tfaction/discussions/1280
- tfcmt: https://github.com/suzuki-shunsuke/tfcmt/discussions/1032

**Comment by ponkio-o** [comment]
https://tech.layerx.co.jp/entry/2023/07/21/175319

**Comment by suzuki-shunsuke** [comment]
https://engineer.retty.me/entry/go-coverage

**Comment by RyoSAKu610** [comment]
I think it'sa  good tool for developing

**Comment by ponkio-o** [comment]
https://speakerdeck.com/ponkio_o/introduction-to-argocd-diff-preview?slide=23
